src/index.js

```diff
@@ -284,6 +285,10 @@ export default class Bili extends EventEmitter {
       throw new BiliError('You must return the options in `extendOptions` method!')
     }
 
+    if (compress && options.pretty) {
+      throw new BiliError('You can not use --pretty with min format')
+    }
+
```

Comment:
no need to throw error since user might run things like `bili --format cjs,umd,umd-min --pretty `, just ignoring `pretty` flag for minified bundle should be good

Reply:
right but they might run `bili --format cjs-min --pretty`, if we ignore silently we should ignore this case too, you OK with that?
But ignoring minified bundle is good idea in total

Reply:
yep, I guess no one wants to prettify minified bundle either 😂

Reply:
Course not, but I'm talking about should we warn them when things like `bili --format cjs-min --pretty` are used or just ignore that silently without any error?

Reply:
maybe log a message via `logger.debug`, adding a waring in docs should be fine for now.